src/server/raft_client.rs

```diff
@@ -568,7 +570,6 @@ where
 
         let cb = ChannelBuilder::new(self.builder.env.clone())
             .stream_initial_window_size(self.builder.cfg.grpc_stream_initial_window_size.0 as i32)
-            .max_send_message_len(self.builder.cfg.max_grpc_send_msg_len)
```

Comment:
So the max message limit is removed in grpc layer,  do we have metrics ( histogram) of the grpc size for this particular Raft batch API?  This is to validate if the size check in RaftClient code works in all the cases and help to diagnostics perf related issues

Reply:
Not yet. I plan to add one once middleware is supported in grpc-rs.

Reply:
Can we remove the entry `max_grpc_send_msg_len` from `Config`? Or mark it as deprecated.

Reply:
No, it's used for controlling the batch size.